views/postStats.pug

```diff
@@ -0,0 +1,21 @@
+html
+  head
+    title Bruins Stats API
+    link(rel='stylesheet', href='/css/styles.css')
+  body
+    header
+      .name Boston Bruin 2019 Stats API
+    table
+        tr
+          th
+            a.link(href=' /stats') Get/stats
+          th
+            a.link(href=' /statsByPlayer') GET/stats/{player}
+          th
+            a.link(href=' /postStats') POST/stats
+          th
+            a.link(href=' /deleteStatsByPlayer') DELETE/stats/{player}
+    p.Route Method: POST/stats
```

Comment:
This route would be impossible for someone to call today. You've provided no information on what data type to send the body as or what fields are required.